CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Winit Contributing Guidelines
+
+## Scope
+
+Winit aims to provide a generic platform abstracting the main graphic platforms (Windows, MacOS, x11, 
+Wayland, Android, iOS and the web platform via Emscripten).
+
+Most platform expose capabilities that cannot be meaningfully transposed to the others. Winit does not
+aim at supporting every single functionaly of every platform, but rather abstract the set of 
+capabilities that is common to all platforms. In this context, APIs exposed in winit can be split into
+different "support levels":
+
+- Tier 1: features which are in the main scope of winit. They are part of the common API of winit, and
+  are taken care of by the maintainers, and them not working correctly is considered a bug in winit.
```

Comment:
s/are taken care of by the maintainers, and them not working correctly is considered a bug in winit./are taken care of by the maintainers. Any part of these features that is not working correctly is considered a bug in winit.